packages/graphql/src/types.ts

```diff
@@ -51,6 +51,7 @@ export interface Context {
     plugins?: Neo4jGraphQLPlugins;
     jwt?: JwtPayload;
     subscriptionsEnabled: boolean;
+    executionContext: Driver | Session | Transaction;
```

Comment:
executionContext is required now, but driver is not required?

Reply:
So `driver` can still be passed into the context, but internally it is all put into `executionContext` in `wrapper.ts` so that we don't have to account for both in the `execute` function. 🙂

Reply:
Ok, thanks for the explanation :)